### PR TITLE
Add legal fees.

### PR DESCRIPTION
**Pull request by benoitdevos**
* Enable to query for legal fee upon LOC creation.

logion-network/logion-internal#893

### CODE REVIEW

File: packages/node-api/integration/Fees.ts [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
     });
     expect(fee).toBeGreaterThanOrEqual(0);
 }
+
+const ONE_LGNT = 1_000_000_000_000_000_000n;
+
+export async function legalFees() {
+    const { api } = await setup();
+    const identityLocFee = await api.fees.estimateLegalFee({ locType: "Identity"});
+    expect(identityLocFee).toEqual(160n * ONE_LGNT);
```
Comment (gdethier) [inline packages/node-api/integration/Fees.ts:17]: You could also use `Currency.nLgnt(...)`.

File: packages/node-api/integration/Fees.ts [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+    const transactionLocFee = await api.fees.estimateLegalFee({ locType: "Transaction"});
+    expect(transactionLocFee).toEqual(2000n * ONE_LGNT);
```
Comment (gdethier) [inline packages/node-api/integration/Fees.ts:19]: You could also use `Currency.nLgnt(...)`.

**Review by gdethier — APPROVED** [review]
:+1:

File: packages/node-api/integration/Fees.ts (hunk 1 above)
Comment (benoitdevos) [reply packages/node-api/integration/Fees.ts]: Done.

**Pull request merged**